RestaurantReservation/Program.cs

```diff
@@ -0,0 +1,147 @@
+﻿namespace RestaurantReservation;
+public class Program
```

Comment:
maybe the start point of the program is better to be in the API project since it is a specific API program